eng/Versions.props

```diff
@@ -20,7 +20,7 @@
     <!--
         When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
     -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
```

Comment:
@wtgodbe - should this be merged

Reply:
@rynowak no. This is a special marker used only in the final stages of a non-preview release.